src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImplTest.java

```diff
@@ -460,11 +537,11 @@ private void assertExpectedOrganisationResponse(OrganisationResponse organisatio
         final int orgIdLength = 7;
         assertThat(organisationResponse).isNotNull();
```

Comment:
prefix test and add verify for when conditions

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/620/commits/c268d76aff9fd43600c21a25a63cedcf73515b00